<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Поиск/Замена</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/srch_32.ico</normaloff>:/srch_32.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLineEdit" name="lineEditSearch"/>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButtonSearch">
     <property name="text">
      <string>Найти (Ctrl + F)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="lineEditReplace"/>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pushButtonReplace">
     <property name="text">
      <string>Заменить (Ctrl + H)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButtonForward">
        <property name="text">
         <string>вперёд</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>409</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkBoxWholeWord">
        <property name="text">
         <string>только целые слова</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" rowspan="2">
       <widget class="QCheckBox" name="checkBoxCaseSens">
        <property name="text">
         <string>с учётом регистра</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioButtonBackward">
        <property name="text">
         <string>назад</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QListWidget" name="listWidget"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
